CoffeeChat/Controller/SocialMediaViewController.swift

```diff
@@ -0,0 +1,202 @@
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        backButton.setImage(UIImage(named: "backArrow"), for: .normal)
+        backButton.addTarget(self, action: #selector(backButtonPressed), for: .touchUpInside)
+        view.addSubview(backButton)
+
+        titleLabel.text = "Connect your social\nmedia?"
+        titleLabel.numberOfLines = 0
+        titleLabel.textAlignment = .center
+        titleLabel.font = ._24CircularStdMedium
+        view.addSubview(titleLabel)
+
+        subtitleLabel.text = "Help your Pear contact you"
+        subtitleLabel.textColor = .greenGray
+        subtitleLabel.font = ._12CircularStdBook
+        view.addSubview(subtitleLabel)
+
+        instagramTextField.backgroundColor = .backgroundWhite
+        instagramTextField.textColor = .black
+        instagramTextField.font = ._20CircularStdBook
+        instagramTextField.placeholder = "@Instagram Handle"
+        instagramTextField.clearButtonMode = .never
+        instagramTextField.addTarget(self, action: #selector(textFieldDidChange(_:)), for: .editingChanged)
+        let instagramImageView = UIImageView(image: UIImage(named: "instagram"))
+        instagramImageView.frame = CGRect(x: 15, y: 15, width: 19, height: 19)
+        instagramTextField.addSubview(instagramImageView)
+        instagramTextField.leftView = UIView(frame: CGRect(x: 0, y: 0, width: 44, height: 49))
+        instagramTextField.leftViewMode = .always
+        instagramTextField.layer.cornerRadius = 8
+        instagramTextField.layer.shadowColor = UIColor(red: 0, green: 0, blue: 0, alpha: 0.15).cgColor
+        instagramTextField.layer.shadowOpacity = 1
+        instagramTextField.layer.shadowRadius = 4
+        instagramTextField.layer.shadowOffset = CGSize(width: 0, height: 2)
+        view.addSubview(instagramTextField)
+
+        facebookTextField.backgroundColor = .backgroundWhite
+        facebookTextField.textColor = .black
+        facebookTextField.font = ._20CircularStdBook
+        facebookTextField.placeholder = "Facebook profile link"
+        facebookTextField.clearButtonMode = .never
+        facebookTextField.addTarget(self, action: #selector(textFieldDidChange(_:)), for: .editingChanged)
+        let facebookImageView = UIImageView(image: UIImage(named: "facebook"))
+        facebookImageView.frame = CGRect(x: 15, y: 15, width: 19, height: 19)
+        facebookTextField.addSubview(facebookImageView)
+        facebookTextField.leftView = UIView(frame: CGRect(x: 0, y: 0, width: 44, height: 49))
+        facebookTextField.leftViewMode = .always
+        facebookTextField.layer.cornerRadius = 8
+        facebookTextField.layer.shadowColor = UIColor(red: 0, green: 0, blue: 0, alpha: 0.15).cgColor
+        facebookTextField.layer.shadowOpacity = 1
+        facebookTextField.layer.shadowRadius = 4
+        facebookTextField.layer.shadowOffset = CGSize(width: 0, height: 2)
+        view.addSubview(facebookTextField)
```

Comment:
with all the repeat code in setting up the button & image, maybe it's worth it to create a function that returns a styled UIButton, given an image name and placeholder text that you pass in? your call, i don't feel too strongly either way, just that `viewDidLoad`is pretty long